lookout/style/format/visualize_predictions.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+from collections import namedtuple
+import os
+import logging
+import sys
+
+import bblfsh
+
+from lookout.core.api.service_data_pb2 import File
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.core import slogging
+from lookout.style.format.features import FeatureExtractor, CLASSES
+from lookout.style.format.model import FormatModel
+
+WHITE = "\033[0m"
+RED = "\033[41m"
+GREEN = "\033[42m"
+ENDC = '\033[m'
+
+
+Misprediction = namedtuple("Misprediction", ["y", "pred", "node"])
```

Comment:
```X, y, nodes = fe.extract_features([file])

    analyzer = FormatModel().load(args.model)
    rules = analyzer._rules_by_lang[args.language]
    y_pred = rules.predict(X)

    mispred = []
    for gt, pred, node in zip(y, y_pred, nodes):
        if gt != pred:
            mispred.append(Misprediction(gt, pred, node))```
I would prefere to avoid it right now - it depends on ` rules.predict` `fe.extract_features` - and they could be changed quite frequently